Nighttime-Operations/Auxiliary-Telescope/AT-Pre/Prepare-for-On-Sky-Operations.rst

```diff
@@ -0,0 +1,112 @@
+
+In a standard night, it will follow the execution of the venting (link) procedure, but it works from any initial state of the system. 
+
+The observers must prepare for on-sky between 2.5 hours and 1 hour before sunset. 
+
+.. _Open-for-On-Sky-Operations-Prerequisites:
+
+Prerequisites
+=============
+
+- Before taking the decision to open, review the weather conditions and :ref:`weather constraints <Nighttime-Operations-Weather-Constraints>` page.
+
+- AuxTel is fully ready to operate and all components are enabled. 
+
+- The :ref:`daytime checkout <AT-Daytime-Checkout-Daytime-Checkout>` has been executed successfully. 
+
+- The telescope has been venting (link to venting once merged) for a considerable amount of time. 
+  If venting wasn't possible, this procedure will take the system from the :ref:`calibration <statesLATISS-Combined-Calibrations-Procedure-LATISS-Combined-Calibrations-Generation-Procedure>` 
+  state or any other initial state to be prepared for on-sky observations. 
+
+.. _Open-for-On-Sky-Operations-Post-Condition:
+
+Post-Condition
+==============
+
+- Telescope is ready for on-sky operations. 
+  
+  Around 15 minutes before the end of the evening nautical twilight, proceed with the :ref:`beginning of the night procedure to center, align and focus the telescope. <AT-On-sky-WEP>`
+
+.. _Open-for-On-Sky-Operations-Procedure-Steps:
+
+Procedure Steps
+===============
+#. Review procedure :ref:`prerequisites <Open-for-On-Sky-Operations-Prerequisites>` and confirm they have been executed. 
+
+#. Announce in the #summit-announce and #summit-auxtel Slack channel that AuxTel is preparing to go on-sky. 
+
+#. Command the telescope and dome to prepare for on-sky observations. 
+
+   Load the ``auxtel/prepare_for/onsky.py`` script from the LOVE ``ATQueue`` panel, under ``AVAILABLE SCRIPTS`` by clicking on the blue icon. 
+
+   In the ``configuring script:onsky`` screen that pops up, leave the ``CONFIG`` box empty. 
+
+   .. figure:: ./_static/PrepareforOnSky_AuxTel.png
+     :name: prepareforonsky_AuxTel
+    
+     Screenshot of LOVE launching ``prepare_for/onsky.py`` script. 
+
+   The script includes the following steps:
+       - Slew telescope to park position. 
+       - If primary mirror cover is open, the script will now close it. 
+         This step is to protect the mirror and avoid particles and dust falling on it when the dome shutter opens.
+       - Move dome to oppose setting Sun to avoid direct sunlight inside the dome that might create thermal issues. 
+       - Open dome main shutter.
+       - Open primary mirror cover and vents. 
+       - Activate AOS open loop corrections.
+       - Enable dome following. 
+
+#. If the wind speed is below 8 m/s, manually open vent gate #3 using the remote controller and turn on the extraction fan power to 20 % from the dome pier.  
+   If wind speed is above or close to 8 m/s, keep vent gates and extraction fan off. 
+
+#. Visually confirm in LOVE displays:
```

Comment:
Same comment about the space when you start a list of items.

Reply:
👍